### PR TITLE
fix: sync attachment files

### PR DESCRIPTION
**Pull request by WindSpiritSR**
This PR can sync file info

image

image

Signed-off-by: WindSpiritSR <simon343riley@gmail.com>

### CODE REVIEW

**Comment by hsluoyz** [comment]
@Kininaru plz review.

File: conversation.go [hunk 1]
```diff
@@ -85,6 +86,17 @@ func (c GoogleGroupConversation) GetAllMessages(client http.Client, removeGmailQ
 		if !ok || len(singleMsgArray0) < 9 {
 			continue
 		}
+		singleMsgArray2, ok := singleMsgArray[2].([]interface{})
+		if ok && len(singleMsgArray2) > 0 {
+			for _, singleFileArray := range singleMsgArray2 {
+				singleFile := singleFileArray.([]interface{})
```
Comment (keainye) [inline conversation.go]: Please make sure that the array `singleFile` is longer than 5 by using an if statement.

**Review by keainye — CHANGES_REQUESTED** [review]
I think this is a very nice PR!
I used to think that attachment files are a part of Gmail quotes, but you found another way to recognize them, that's really nice.

File: conversation.go (hunk 1 above)
Comment (Zxilly) [reply conversation.go]: @Kininaru that->than

Comment (keainye) [reply conversation.go]: @Zxilly Thanks, corrected.

Comment (WindSpiritSR) [reply conversation.go]: @Kininaru Fixed

**Comment by keainye** [comment]
LGTM

**Pull request merged**